js_modules/dagster-ui/packages/ui-core/src/runs/RunStatusPez.tsx

```diff
@@ -31,35 +31,48 @@ interface ListProps {
   fade: boolean;
   jobName: string;
   runs: RunTimeFragment[];
+  forceCount?: number;
 }
 
 export const RunStatusPezList = (props: ListProps) => {
-  const {fade, jobName, runs} = props;
+  const {fade, jobName, runs, forceCount} = props;
   const count = runs.length;
   const countForStep = Math.max(MIN_OPACITY_STEPS, count);
   const step = (MAX_OPACITY - MIN_OPACITY) / countForStep;
+
+  let items: (RunTimeFragment | null)[] = [...runs];
+  if (forceCount) {
+    if (forceCount > items.length) {
+      items.unshift(...Array(forceCount - items.length).fill(null));
+    } else {
+      items = items.slice(0, forceCount);
+    }
+  }
```

Comment:
A new change to `RunStatusPezList`. @braunjj asks that the pez list occupy the same amount of space even if there are fewer than N items in the list, so this allows us to fill the space with placeholder pez items.